src/api/routes/profileRouter.js

```diff
@@ -0,0 +1,15 @@
+import { Router } from 'express';
+import profileController from '../controllers/profileController';
+import upload from '../../middlewares/upload';
+import validateInputs from '../../middlewares/validations/body.inputs';
+
+
+const profileRouter = new Router();
+const fields = ['country', 'lastName', 'firstName', 'address', 'gender', 'phoneNumber', 'bio', 'profileImage', 'facebook', 'twitter'];
+
+profileRouter.patch('/', upload.single('profileImage'), validateInputs(true, 'profile', fields),
+  profileController.update);
+profileRouter.get('/', profileController.getUserProfile);
```

Comment:
@alainburindi I thought we agreed to merge `getUserProfile` and `read` into one route for getting a user's profile.